packages/components/src/components/DaScroll.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div
+    class="scroll__switch"
+    :class="[showed ? 'shown' : '']"
+    @click="scrollTop"
+  >
+    <svgicon name="arrow" class="scroll__switch__icon" />
+  </div>
+</template>
+
+<script>
+export default {
+  name: "DaScroll",
+
+  data() {
+    return {
+      showed: false,
+    };
+  },
+
+  methods: {
+    scrollTop() {
+      window.scrollTo({ top: 0, behavior: "smooth" });
+    },
+  },
+
+  created() {
+    this.scroll = (event) => {
+      this.showed =
+        document.documentElement.scrollTop >
+        document.documentElement.scrollHeight / 3;
+    };
+    this.scroll = this.scroll.bind(this);
+  },
+
+  mounted() {
+    import("../../icons/arrow");
+    window.addEventListener("scroll", this.scroll);
+  },
+};
+</script>
+
+<style>
+.scroll__switch {
+  height: 48px;
+  width: 48px;
+  border-radius: 100%;
+  bottom: 32px;
+  right: 32px;
+  opacity: 0;
+  transition: 0.35s;
+  transform: scale(0.8);
+  position: fixed;
+  background: var(--theme-primary);
+  visibility: hidden;
+
+  &:hover {
+    opacity: 1 !important;
```

Comment:
Please don't use `!important`, it's not a good practice. It's better to increase specificity to make sure this rule is applied